src/adlist.h

```diff
@@ -88,6 +88,13 @@ void listRewindTail(list *list, listIter *li);
 void listRotateTailToHead(list *list);
 void listRotateHeadToTail(list *list);
 void listJoin(list *l, list *o);
+static inline void listInitEmbeddedNode(listNode * node, void *parentStruct) {
```

Comment:
```suggestion
static inline void listInitNode(listNode *node, void *value) {
```
The nodes need not necessarily be embedded, you could have just allocated memory separately, so I don't think it's any less clear by keeping it generic.

Reply:
yup, will change